packages/laconia-event/src/DynamoDbStreamRecord.js

```diff
@@ -1,18 +1,36 @@
-const AWS = require("aws-sdk");
+require("aws-sdk/clients/dynamodb");
+const converter = require("aws-sdk/lib/dynamodb/converter");
 
 module.exports = class DynamoDbStreamRecord {
   constructor(data) {
     this.data = data;
```

Comment:
Maybe you could use destructuring here:
```javascript
this.data = data || {};
```
I did some test here and if you call `converter.unmarshall(undefined)` (same for null and {}) it returns `{}`

Your result will look like this:
```javascript
return {
  eventName: this.eventName,
  StreamViewType: data.StreamViewType,
  Keys: converter.unmarshall(data.Keys),
  NewImage: converter.unmarshall(data.NewImage),
  OldImage: converter.unmarshall(data.OldImage)
};
```

Reply:
Perfect will implement that